patches/react-native/details.md

```diff
@@ -182,4 +182,11 @@
 - Reason: This patch updates Yoga to correctly update the subtrees of `display: contents` nodes so that they are in sync with their React Native counterparts.
 - Upstream PR/issue: https://github.com/facebook/react-native/pull/52530
 - E/App issue: https://github.com/Expensify/App/issues/65268
-- PR introducing patch: [#65925](https://github.com/Expensify/App/pull/65925)
\ No newline at end of file
+- PR introducing patch: [#65925](https://github.com/Expensify/App/pull/65925)
+
+### [react-native+0.79.2+026+fix-nested-modals.patch](react-native+0.79.2+026+fix-nested-modals.patch)
+
+- Reason: Fixes an issue where deeply nested modals fail to appear, and the app becomes unresponsive after closing them. The patch ensures modals are always presented from the topmost currently visible view controller, resolving incorrect modal stacking behavior on iOS.
+- Upstream PR/issue: https://github.com/facebook/react-native/issues/52790
+- E/App issue: https://github.com/Expensify/App/issues/66247
```

Comment:
E/App issue should be an issue created specifically to track this patch progress (see [this](https://github.com/Expensify/App/blob/135002461d3072de0363d7ce000c6b592cb0e2f4/contributingGuides/PATCHES.md?plain=1#L92) line)

Issue example: https://github.com/Expensify/App/issues/66231